src/data/services/ReleaseNotesService.ts

```diff
@@ -117,17 +167,17 @@ export class ReleaseNotesService {
         });
     }
 
-    private async getPullRequestWorkItems(gitClient: GitRestClient, repositoryId: string, pullRequestId: number): Promise<number[]> {
-        const workItemRefs = await gitClient.getPullRequestWorkItemRefs(repositoryId, pullRequestId);
-        return workItemRefs.map(w => Number(w.url.split('/').pop()));
-    }
-
-    private async getCommentsWorkItems(gitClient: GitRestClient, repositoryId: string, pullRequestId: number): Promise<number[]> {
+    private async getCommitsWorkItems(gitClient: GitRestClient, repositoryId: string, pullRequestId: number): Promise<number[]> {
         const commitRefs = await gitClient.getPullRequestCommits(repositoryId, pullRequestId);
 
-        return commitRefs.map(c => {
-            const match = /\b#[0-9]+\b/.exec(c.comment);
-            return match ? match.map(x => Number(x)) : [];
-        }).reduce((c, a) => c.concat(a), []);
+        return commitRefs
+            .map(c => this.getWorkItemNumbersFromText(c.comment))
+            .reduce((c, a) => c.concat(a), []);
+    }
+
+    private getWorkItemNumbersFromText(text: string) {
+        const regex = /\d+/g;
```

Comment:
A hash (`#`) is required to correctly reference a work item in commit text (that's how ADO understands it's a reference).
Also, such text must be separated from the rest of text (\b for break), thus a better pattern is `/\b#\d+\b/`